appmonitoring/ts/src/CertificateManager.ts

```diff
@@ -131,7 +131,7 @@ export class CertificateManager {
         }
     }
 
-    public async PatchSecretStore(operationId: string, kubeConfig: k8s.KubeConfig, certificate: WebhookCertData) {
+    public async UpdateSecretStore(operationId: string, kubeConfig: k8s.KubeConfig, certificate: WebhookCertData) {
```

Comment:
Async methods that return "nothing" should return Promise<void>, let's update it here and for other async methods in this file while we are at it.